### PR TITLE
Add some logging

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
This is an automated comment for commit 6b97f413da64041d518cb36f58ee39280cdce659 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/49690/6b97f413da64041d518cb36f58ee39280cdce659/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: src/Interpreters/Cache/LRUFileCachePriority.cpp [hunk 1]
```diff
@@ -138,28 +147,26 @@ void LRUFileCachePriority::iterate(IterateFunc && func, const CacheGuard::Lock &
...
-LRUFileCachePriority::Iterator LRUFileCachePriority::LRUFileCacheIterator::remove(const CacheGuard::Lock &)
+LRUFileCachePriority::Iterator
+LRUFileCachePriority::LRUFileCacheIterator::remove(const CacheGuard::Lock &)
 {
-    return std::make_shared<LRUFileCacheIterator>(cache_priority, cache_priority->remove(queue_iter));
+    return std::make_shared<LRUFileCacheIterator>(
+        cache_priority, cache_priority->remove(queue_iter));
 }
 
 void LRUFileCachePriority::LRUFileCacheIterator::annul()
```
Comment (alesapin) [inline src/Interpreters/Cache/LRUFileCachePriority.cpp:157]: what is annul?

Comment (kssenii) [reply src/Interpreters/Cache/LRUFileCachePriority.cpp:157]: Screenshot 2023-05-09 at 19 35 15

**Review by alexey-milovidov — APPROVED** [review]

**Comment by kssenii** [comment]
> Stress test (ubsan) — Killed by signal 

not related to changes, will fix in a separate pr, fixing this one now

**Pull request merged**